boards/arm/adafruit_itsybitsy_m4_express/adafruit_itsybitsy_m4_express.dts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <atmel/samd5xx19.dtsi>
+
+/ {
+	model = "Adafruit ItsyBitsy M4 Express";
+	compatible = "adafruit,itsybitsy-m4-express", "atmel,atsamd51g19a", "atmel,samd51";
```

Comment:
This should just be the board here (drop "atmel,atsamd51g19a", "atmel,samd51")

Reply:
Done